api/v1alpha1/oadp_types.go

```diff
@@ -180,6 +180,19 @@ type DataMover struct {
 	// the number of batched volumeSnapshotRestores that can be inProgress at once, default value is 10
 	// +optional
 	MaxConcurrentRestoreVolumes string `json:"maxConcurrentRestoreVolumes,omitempty"`
+	// defines configurations for VolSync options
+	// +optional
+	MoverConfig *MoverConfig `json:"moverConfig,omitempty"`
```

Comment:
IMO `MoverConfig` may sound like configuration parameters for datamover, I believe our aim is to set options for the Volume, so maybe `VolumeOptions` or something on similar lines. wdyt ?